static_src/actions/login_actions.js

```diff
@@ -5,13 +5,40 @@
  */
 
 import AppDispatcher from '../dispatcher.js';
+import cfApi from '../util/cf_api';
 import { loginActionTypes } from '../constants';
 
-export default {
-  receivedStatus(status) {
+const loginActions = {
+  fetchStatus() {
+    AppDispatcher.handleViewAction({
+      type: loginActionTypes.FETCH_STATUS,
+      status
```

Comment:
should this be `authStatus`? (as a reactjs newbie, let me know so i can understand)